CHANGELOG.md

```diff
@@ -61,7 +64,7 @@ Before releasing:
 
 - Overhauled the `competition` module with more straightforward getters for competition state. (#38) (**Breaking Change**)
 - LLEMU-related macros have been prefixed with `llemu_` (e.g. `llemu_println`). (**Breaking Change**) (#30)
-- Added `Debug`, `Copy`, and `Clone` derives for common structs (#37)
+- Added `Debug`, `Copy`, and `Clone` derives for common structs (#37, #70)
```

Comment:
This should be added to the unreleased section instead of an old already released version.